pipelines/gitops-pull-request-rhtap/gitops-pull-request.yaml

```diff
@@ -71,6 +82,10 @@ spec:
           value: $(params.ec-tuf-mirror)
       runAfter:
         - gather-deploy-images
+      when:
+        - input: $(tasks.gather-deploy-images.results.NOTHING_TO_VERIFY)
+          operator: notin
+          values: ["true"]
```

Comment:
Is it possible to have something like
```
 - input: $(tasks.gather-deploy-images.results.IMAGES_TO_VERIFY)
   operator: notin
   values: [""]
```
instead of a new result?

Reply:
This would probably work (after fixing some trailing newlines)
```yaml
 - input: $(tasks.gather-deploy-images.results.IMAGES_TO_VERIFY)
   operator: notin
   values: ['{"components":[]}']
```
That feels a bit fragile, like we're exposing too much of the implementation details. If the output wasn't in `jq -c` format, the JSON could be formatted a bit differently

Reply:
The above is definitely fragile, but we can check `/tmp/all-images.txt` before invoking `jq` and exit with empty result earlier.

Reply:
good point, that should work

Reply:
Adjusted as suggested :+1: